internal/runners/revert/revert.go

```diff
@@ -53,32 +53,39 @@ func New(prime primeable) *Revert {
 	}
 }
 
+const headCommitID = "HEAD"
+
 func (r *Revert) Run(params *Params) (rerr error) {
 	defer rationalizeError(&rerr)
 
 	if r.project == nil {
 		return locale.NewInputError("err_no_project")
 	}
-	if !strfmt.IsUUID(params.CommitID) {
+
+	commitID := params.CommitID
+	if !strfmt.IsUUID(commitID) && commitID != headCommitID {
```

Comment:
I think here and the other equality check with the constant should be case insensitive. I think `strings.EqualFold` will get you there.